### PR TITLE
Add anchor-position polyfill update

### PR DESCRIPTION
**Pull request by jamesnw**
## Description
An update on the Anchor Position polyfill.

## Related Issue(s)
#737

https://deploy-preview-739--oddleventy.netlify.app/2024/11/12/anchor-position-polyfill-2/

### CODE REVIEW

File: content/blog/2024/anchor-position-polyfill-2.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+---
+title: Continued updates to the Anchor Position Polyfill
```
Comment (jgerigmeyer) [inline content/blog/2024/anchor-position-polyfill-2.md:2]: Should we standardize on either `Anchor Position Polyfill` or `Anchor Positioning Polyfill`?

File: content/blog/2024/anchor-position-polyfill-2.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+sub: Performance and
```
Comment (jgerigmeyer) [inline content/blog/2024/anchor-position-polyfill-2.md:3]: ...and...and what?!? 😮

File: content/blog/2024/anchor-position-polyfill-2.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  Here's a summary of the latest updates.
+---
+{% import 'embed.macros.njk' as embed %}
+
+A lot has happened with Anchor Positioning since our [last update](/2024/07/02/anchor-position-polyfill/).
+
+With our [polyfill](https://github.com/oddbird/css-anchor-positioning), we've
+made a lot of progress in our goal of catching up to the spec, and based on the
+feedback we're hearing, people are finding it covers enough of their needs to be
+able to start using anchor positioning in production.
```
Comment (jgerigmeyer) [inline content/blog/2024/anchor-position-polyfill-2.md:29]: "With our polyfill" sounds a bit awkward as a starting phrase. I'm not tied to this, but maybe something like:

```suggestion
We've made a lot of progress in our goal of catching up our
[polyfill](https://github.com/oddbird/css-anchor-positioning) to the spec, and
based on the feedback we're hearing, people are finding it covers enough of
their needs to be able to start using anchor positioning in production.
```

File: content/blog/2024/anchor-position-polyfill-2.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+defining which pseudo-elements can be anchors was recently updated from
+`part-like` to `fully styleable tree-abiding pseudo-element`. This means that
+`::file-selector-button` and `::details-content` are not anchors.
+`::-webkit-slider-thumb` is not likely to fit this criteria either, which is a
+shame as it's been [extremely](https://codepen.io/jamessw/pen/KKLMJKm)
+[useful](https://codepen.io/jamessw/pen/ZENpWao) for examples.
+
+{% endcallout %}
+
+## anchor-scope support
```
Comment (jgerigmeyer) [inline content/blog/2024/anchor-position-polyfill-2.md:49]:
```suggestion
## Support for scoping anchor names
```

File: content/blog/2024/anchor-position-polyfill-2.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+is extremely useful when working with lists of elements that each have their own
+anchor elements.
+
+In this example, anchoring is applied with shared `.target` and `.anchor` rules,
+meaning there are multiple elements with `anchor-name: --list-item` set. By
+default, the last element with that rule will be the anchor.
+
+But that's not what we want. We want the anchoring element within the same list
+item to be our anchor. To do that, set `anchor-scope: --list-item` on
+the `li` element, to limit the search for an anchoring element.
```
Comment (jgerigmeyer) [inline content/blog/2024/anchor-position-polyfill-2.md:61]:
```suggestion
the `li` element to limit the search for an anchoring element.
```

File: content/blog/2024/anchor-position-polyfill-2.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+
+Note that Chromium released support for `anchor-scope` in version 131, but since
+the polyfill doesn't support partial polyfilling, this demo will only work in
+Chromium before 125, after 131, or in a non-Chromium browser.
+
+## Position fallbacks
+
+What happens when an anchored element doesn't fit where it would go? We
+implemented the ["Overflow
+Management"](https://drafts.csswg.org/css-anchor-position/#fallback) portion of
```
Comment (jgerigmeyer) [inline content/blog/2024/anchor-position-polyfill-2.md:77]:
```suggestion
implemented the [Overflow
Management](https://drafts.csswg.org/css-anchor-position/#fallback) portion of
```

File: content/blog/2024/anchor-position-polyfill-2.md [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+  id='abrZPQY',
+  title='Fallbacks with `position-try`',
+  user='jamessw'
+) }}
+
+## Performance improvements
+
+v0.3.0 was a performance-focused release. It turns out that applying anchor
+positioning is a fairly intensive process, which is one of the reasons why it's
+getting moved from the realm of JavaScript into the native browser.
```
Comment (jgerigmeyer) [inline content/blog/2024/anchor-position-polyfill-2.md:94]:
```suggestion
[v0.3.0](https://github.com/oddbird/css-anchor-positioning/releases/tag/v0.3.0)
was a performance-focused release. It turns out that applying anchor positioning
is a fairly intensive process, which is one of the reasons why it's getting
moved from the realm of JavaScript into the native browser.
```

File: content/blog/2024/anchor-position-polyfill-2.md [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+
+[@marchbox](https://github.com/marchbox) contributed a huge win that allows you
+to apply the polyfill to just the css files that impact anchor positioning. By
```
Comment (jgerigmeyer) [inline content/blog/2024/anchor-position-polyfill-2.md:97]:
```suggestion
to apply the polyfill to just the styles that impact anchor positioning. By
```

File: content/blog/2024/anchor-position-polyfill-2.md [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+default, the polyfill parses all of your CSS, which is likely more than you
+need. You're now able to specify exactly which CSS stylesheets contain rules
+that impact how you use anchor positioning, so not all CSS has to be parsed.
+
+Native implementations are able to calculate anchor positioning much more efficiently.
```
Comment (jgerigmeyer) [inline content/blog/2024/anchor-position-polyfill-2.md:102]: I'd probably flesh this out a bit or remove it? It seems related, but a bit tangential.

```suggestion
```

File: content/blog/2024/anchor-position-polyfill-2.md [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+
+## Coming soon
+
+What are you excited for with anchor positioning? We'd love to implement
+[`position-area`](https://github.com/oddbird/css-anchor-positioning/issues/181)
+and apply the polyfill to [dynamic
+elements](https://github.com/oddbird/css-anchor-positioning/issues/91).
+
+If you want to start using anchor positioning, there's a few ways you can help.
```
Comment (jgerigmeyer) [inline content/blog/2024/anchor-position-polyfill-2.md:111]:
```suggestion
If you want to start using anchor positioning, there are a few ways you can help.
```

File: content/blog/2024/anchor-position-polyfill-2.md [hunk 11]
```diff
@@ -0,0 +1,119 @@
...
+You can take a cue from the contributors listed in this article and take on a
+feature or fix a bug. You can also fund our work by becoming a sponsor.
```
Comment (jgerigmeyer) [inline content/blog/2024/anchor-position-polyfill-2.md:113]:
```suggestion
feature or fix a bug. You can also fund our work by [becoming a
sponsor](https://opencollective.com/oddbird-open-source).
```

File: content/blog/2024/anchor-position-polyfill-2.md [hunk 12]
```diff
@@ -0,0 +1,119 @@
...
+
+Another way is to [hire us](/services/). If your company wants to use the Anchor
+Positioning Polyfill, hire us to complete a needed feature, or to finish the
+polyfill. Our client work also helps fund our open source work like this
+polyfill, so [get in touch](https://www.oddbird.net/contact/) with us if you
+have any web development needs.
```
Comment (jgerigmeyer) [inline content/blog/2024/anchor-position-polyfill-2.md:119]:
```suggestion
Another way to support this work is to [hire us](/services/). If your company
wants to use the Anchor Positioning Polyfill, hire us to complete a needed
feature, or to finish the polyfill. Our client work also helps fund our open
source work like this polyfill, so [get in
touch](https://www.oddbird.net/contact/) with us if you have any web development
needs.
```

**Review by jgerigmeyer — APPROVED** [review]
👏

File: content/blog/2024/anchor-position-polyfill-2.md (hunk 1 above)
Comment (jamesnw) [reply content/blog/2024/anchor-position-polyfill-2.md]: Anchor Positioning is in the spec, so I'll standardize on that.

File: content/blog/2024/anchor-position-polyfill-2.md (hunk 2 above)
Comment (jamesnw) [reply content/blog/2024/anchor-position-polyfill-2.md]: Puppies, probably. Updated.

**Review by jgerigmeyer — APPROVED** [review]

File: content/blog/2024/anchor-position-polyfill-2.md [hunk 13]
```diff
@@ -0,0 +1,130 @@
...
+Along with changes to the spec and Chromium's continued work towards
+implementing the spec, there's been a lot of interest in using anchor
+positioning. In fact, the State of CSS survey identified anchor positioning as
+the [most desired
+feature](https://2024.stateofcss.com/en-US/usage/#css_interoperability_features)
+that you can't yet use because of browser
+incompatibilities.
+
+And while it's true that anchor positioning is only implemented in Chromium
+browsers, our team at OddBird have made a lot of progress in our goal of
```
Comment (jgerigmeyer) [inline content/blog/2024/anchor-position-polyfill-2.md:35]:
```suggestion
browsers, our team at OddBird has made a lot of progress in our goal of
```

**Review by SondraE — APPROVED** [review]
- In the title , capitalize the "U" in "Updates". 
- In the title could you replace "Continued" with "Exciting" or something more descriptive? Or maybe something intriguing like "Updates to the Most Desired CSS Feature You Can't Use Yet"
- Add comma after scope in subtitle
- I wonder if the Coming Soon paragraph could get bullet points like what we wrote at the bottom of the Anchor Positioning README so that the ways to support the work are more skim-able.

**Pull request merged**